qcodes/instrument_drivers/AlazarTech/ATS9360.py

```diff
@@ -405,4 +403,4 @@ def _set_trigger_holdoff(self, value: bool) -> None:
             # otherwise
             disable_mask = ~np.uint32(1 << 26)
             new_value = current_value & disable_mask
-        self._write_register(58, new_value)
+        self._write_register(58, new_value)
```

Comment:
You have deleted a new line here. Please add that back to make sure the file matches our normal style